src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -426,8 +353,8 @@ static AnimationInfoS sAnimationInfo[] = {
  * @param maxFrames Maximum number of frames the effect will last. Actual lifetime will be between 'maxFrames / 3' and
  * 'maxFrames'
  */
-EnGoEffect* EnGo_InitSteam(EnGoEffect effect[ENGO_OTHER_EFFECT_COUNT], Vec3f pos, Vec3f accel, Vec3f velocity,
-                           f32 scale, f32 deltaScale, s32 maxFrames) {
+EnGoEffect* EnGo_InitSteam(EnGoEffect effect[], Vec3f pos, Vec3f accel, Vec3f velocity, f32 scale, f32 deltaScale,
```

Comment:
I don't think it's very useful to retain the [] without an explicit dimension, since that's pretty much the only thing that the compiler can get out of this notation.

Reply:
well at least it tells the user that this function is meant to take an array, so it isn't completely useless

Reply:
Ah, actually I was going to bring up: I removed these because there was a mismatch between these and the function declarations above which actually generate a lot of warnings for this file (which I'm not sure why it wasn't picked up by Jenkins). But just removing the headings above and having the size was fine, so I've brought them back